src/app/GitUI/NBugReports/BugReportInvoker.cs

```diff
@@ -366,6 +367,17 @@ static void ShowGitRepo(Form? ownerForm, string? workingDir)
                         });
                 }
             }
+
+            static string ReplaceQuotes(string command)
```

Comment:
A comment explaining what the method does with some examples would go a long way